NAA_Analysis.py

```diff
@@ -0,0 +1,245 @@
+        if isotope == var:
+            isotope_info = (ia.natural_isotope_list[i].mass_number,
+                            ia.natural_isotope_list[i].isotopic_abundance,
+                            ia.natural_isotope_list[i].cross_section)
+            flag = True
+            break
+
+    if flag is False:
+        isotope_info = [0, 0, 0]
+
+    return(flag, isotope_info[0], isotope_info[1], isotope_info[2])
+
+
+def element_fraction(d_per_s, isotope, mass_number, I_A, cross_section,
+                     flux, delta, t_irrad, weight):
+    N_0 = 6.022 * 10**23
+    """cross section, decay constant, isotopic abundance, atomic weight
+       will be obtained from an isotope class"""
+    first = d_per_s * mass_number / (N_0 * weight * I_A)
+    second = first * isotope.decay_constant**(-1)
+    third = second * np.exp(delta * isotope.decay_constant)
+    fourth = third / (flux * (cross_section * 10**-24) * t_irrad)
+    return fourth
+
+
+def sample_questions():
+    weight = input('\n''Enter sample mass in grams.' '\n' '-->')
+    weight_value = False
+    while weight_value is False:
+        try:
+            weight = float(weight)
+            weight_value = True
+        except ValueError:
+            print('\n''Please enter a number for mass.''\n')
+            weight = input('Enter sample mass in grams.' '\n' '-->')
+
+    flux = input('\n''Enter reactor thermal neutron flux [n/cm2s].' '\n' '-->')
+    flux_value = False
+    while flux_value is False:
+        try:
+            flux = float(flux)
+            flux_value = True
+        except ValueError:
+            print('\n''Please enter a number for flux.''\n')
+            flux = input('Enter reactor thermal neutron flux [n/cm2s].' '\n'
+                         '-->')
+
+    delta_t = input('\n'"Enter the time (in seconds) between the "
+                    "irradiation and the measurement." '\n' '-->')
+    delta_t_value = False
+    while delta_t_value is False:
+        try:
+            delta_t = float(delta_t)
+            delta_t_value = True
+        except ValueError:
+            print('\n''Please enter a number for time.''\n')
+            delta_t = input("Enter the time (in seconds) between the "
+                            "irradiation and the measurement." '\n' '-->')
+
+    t_irrad = input('\n''Enter the length of irradiation in seconds.'
+                    '\n' '-->')
+    t_irrad_value = False
+    while t_irrad_value is False:
+        try:
+            t_irrad = float(t_irrad)
+            t_irrad_value = True
+        except ValueError:
+            print('\n''Please enter a number for irradiation time.''\n')
+            t_irrad = input('Enter the length of irradiation in seconds.'
+                            '\n' '-->')
+    return weight, flux, delta_t, t_irrad
+
+
+def acquire_measurement():
+    measurements = []
+    dir_path = os.getcwd()
+    for file in os.listdir(dir_path):
+        if file.endswith(".Spe"):
```

Comment:
I've noticed here and in a couple places that the code depends on the case of the `Spe` extension. You can avoid this by using string method `lower()`:
```
if file.lower().endswith(".spe"):
```
Then it will work whether the extension is `Spe` or `spe` or `SPE`.